test/parallel/test-buffer-writedouble.js

```diff
@@ -70,6 +70,10 @@ buffer.writeDoubleLE(NaN, 8);
 assert.ok(buffer.equals(new Uint8Array([
   0x7F, 0xF8, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
   0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xF8, 0x7F
+// Mips processors use a different NaN.
+])) || buffer.equals(new Uint8Array([
```

Comment:
Can we detect running on mips and only assert the alternative value if so?

Reply:
Done.